pyscriptjs/src/pyodide.ts

```diff
@@ -91,11 +90,25 @@ export class PyodideRuntime extends Runtime {
     }
 
     async loadFromFile(path: string): Promise<void> {
-        const filename = getLastPath(path);
+        const pathArr = path.split('/');
+        const filename = pathArr.pop();
+        for(let i=0; i<pathArr.length; i++)
+        {
+            const eachPath = pathArr.slice(0, i+1).join('/');
+            const {exists, parentExists} = this.interpreter.FS.analyzePath(eachPath);
+            if (!parentExists) {
+                throw Error(`parent path for ${eachPath} doesn't exist, cannot create it.`)
+            }
```

Comment:
does this mean that if I do this:
```
paths = ['a/b/c/foo.py']
```
it won't work because I need to create each directory manually?
Any reason for why not creating all the needed directories?

Reply:
No, so, the parent of `a` in this case is nothing but the root of the filesystem which will always exist.
So we won't enter this condition. Ideally, there is no need of checking this, since the manner in which we proceed, it will make sure that `a` exists before checking for `b` and so on...

Reply:
so this is basically an assertion? Then better to make it clear, because it wasn't clear to me :).
Something like:
`throw new Error('INTERNAL ERROR! cannot create {path}, this should never happen')`
or something like that.

Reply:
yeah, guess I should change it, thanks!